src/data/complex/exponential.lean

```diff
@@ -1159,23 +1159,31 @@ by rw ← of_real_inj; simp [sinh_three_mul]
 
 open is_absolute_value
 
+lemma quadratic_le_exp_of_nonneg {x : ℝ} (hx : 0 ≤ x) : 1 + x + x ^ 2 / 2 ≤ exp x :=
+calc 1 + x + x ^ 2 / 2
+    = ∑ i in range 3, x ^ i / i! : by simp [finset.sum_range_succ]
+... ≤ lim (⟨_, is_cau_seq_re (exp' x)⟩ : cau_seq ℝ has_abs.abs) :
+  begin
+    refine le_lim (cau_seq.le_of_exists ⟨3, λ j hj, _⟩),
+    simp only [exp', const_apply, mk_to_fun, re_sum],
+    norm_cast,
+    rw [← nat.add_sub_of_le hj, finset.sum_range_add],
+    refine le_add_of_nonneg_right (sum_nonneg (λ i hi, _)),
+    positivity,
+  end
+... = exp x : by rw [exp, complex.exp, ← cau_seq_re, lim_re]
```

Comment:
Can we add the more general result that `∑ i in range n, x ^ i / i! ≤ exp x` and get this as a special case?